Gopkg.lock

```diff
@@ -3,108 +3,151 @@
 
 [[projects]]
   branch = "master"
+  digest = "1:315c5f2f60c76d89b871c73f9bd5fe689cad96597afd50fb9992228ef80bdd34"
   name = "github.com/alecthomas/template"
   packages = [
     ".",
-    "parse"
+    "parse",
   ]
+  pruneopts = "UT"
   revision = "a0175ee3bccc567396460bf5acd36800cb10c49c"
 
 [[projects]]
   branch = "master"
+  digest = "1:c198fdc381e898e8fb62b8eb62758195091c313ad18e52a3067366e1dda2fb3c"
   name = "github.com/alecthomas/units"
   packages = ["."]
+  pruneopts = "UT"
   revision = "2efee857e7cfd4f3d0138cc3cbb1b4966962b93a"
 
 [[projects]]
   branch = "master"
+  digest = "1:d6afaeed1502aa28e80a4ed0981d570ad91b2579193404256ce672ed0a609e0d"
   name = "github.com/beorn7/perks"
   packages = ["quantile"]
+  pruneopts = "UT"
   revision = "3a771d992973f24aa725d07868b467d1ddfceafb"
 
 [[projects]]
+  digest = "1:97df918963298c287643883209a2c3f642e6593379f97ab400c2a2e219ab647d"
   name = "github.com/golang/protobuf"
   packages = ["proto"]
-  revision = "925541529c1fa6821df4e44ce2723319eb2be768"
-  version = "v1.0.0"
+  pruneopts = "UT"
+  revision = "aa810b61a9c79d51363740d207bb46cf8e620ed5"
+  version = "v1.2.0"
 
 [[projects]]
   branch = "master"
+  digest = "1:8f81b9df5085144dafc26d688d5f194ef78d23990213cea912e56196691cb596"
   name = "github.com/hazelcast/hazelcast-go-client"
   packages = [
     ".",
     "config",
+    "config/property",
     "core",
     "internal",
-    "internal/common",
-    "internal/common/collection",
-    "internal/flake_id",
-    "internal/predicates",
-    "internal/protocol",
+    "internal/aggregation",
+    "internal/discovery",
+    "internal/flakeid",
+    "internal/murmur",
+    "internal/precond",
+    "internal/predicate",
+    "internal/projection",
+    "internal/proto",
+    "internal/proto/bufutil",
+    "internal/reliabletopic",
     "internal/serialization",
+    "internal/serialization/bufferutil",
     "internal/serialization/classdef",
+    "internal/util/colutil",
+    "internal/util/iputil",
+    "internal/util/timeutil",
+    "internal/util/versionutil",
+    "security",
     "serialization",
-    "serialization/classdef"
+    "serialization/classdef",
   ]
-  revision = "57b5e47611af979ddc50b1f628bb38c7d44db0d4"
+  pruneopts = "UT"
+  revision = "3560be65ca6037e3adf8cf5cd222dabf0f980530"
```

Comment:
Set revision to `4e0c3ccf8841745e674e2419f3dcb20de0a60098` please. This contains latest fixes for connections not being closed.